### DISCUSSION TITLE
1.16.5

### DISCUSSION THREAD

**Discussion by PluginDownloader** [Geolykt/EnchantmentsPlus#133] [Q&A] [answered]
Hello. I don't really know how to make it work on 1.16.5. I just keep getting "an internal error has occurred" But its still shows green when i type /plugin. Really love this Plugin and would appreciate if you guys could help me out!

**Comment by PluginDownloader** [comment]
Forgot to mention, when i download EnchantmentsPlus it doesnt show on Plugin. but zenenchantments does

**Reply by Geolykt** [reply]
Where exactly did you download it from?

**Comment by Geolykt** [comment]
Could you show us an excerpt of the logs if possible? Also I would like to know what is printed if you do `/ench version`

**Reply by PluginDownloader** [reply]
The Version is 1.16.2, but when i downloaded it said it was for 1.16.5 o:  and here are the logs, idk if its the correct one tho. i am big novice
https://pastebin.com/ebBk1Q5g
i just copied everything :D!
If thats not 1.16.5 version what is o:

**Reply by Geolykt** [reply]
Download <https://github.com/Geolykt/EnchantmentsPlus/releases/tag/v4.0.5> instead - or if you do not like fiddling with config files to get the best 1.16 experience use  <https://github.com/Geolykt/EnchantmentsPlus/releases/tag/v3.1.6>, although I recommend against that because that is an older version.

What you are using right now is Zenchantments, perhaps NMS-less Zenchantments - regardless those are older versions of this plugin and do not fully support 1.16.X

**Reply by Geolykt** [reply]
Ah yeah, 1.6.2 is NMS-ful Zenchantments for minecraft version 1.14 - which is completely incompatible with 1.16 because it is using NMS (and there is an enchantment parsing bug when used in later versions, which is the core reason this fork exists at all)

**Reply by PluginDownloader** [reply]
Fiddling with configs isn't my thing, big novice :D! So I downloaded the second one. but it doesn't seem to be working thought, wont even pop on /plugin. Am i doing something wrong haha

**Reply by PluginDownloader** [reply]
I went to Magiccompat it and says meant for 
# default configurations as of Version 3.0.0 of the Enchantments+ Plugin, meant for use for minecraft 1.16.4
idk if thats important tho

**Reply by PluginDownloader** [reply]
Still confused why does, Old version like Zenenchantments show but not this one

**Reply by PluginDownloader** [reply]
Sorry for spam replying, i tried 4.0.5 as well but nothing :/

**Reply by PluginDownloader** [reply]
https://pastebin.com/HLWDLdjR
Idk if this showed in the Logs i sent u


![LLL](https://user-images.githubusercontent.com/118046006/201512669-2ca1be83-baab-47dd-a3d1-fc4cc73303df.PNG)
now i am just trying put all of  them there haha, It wont even make a folder for the plugin

**Reply by Geolykt** [reply]
Just use the latest one (4.0.5). And install Java 16 or higher (I believe 1.16.5 will run under Java 16 at maximum though), which is required in order to run EnchantmentsPlus - I know that it is an extra hassle for server owners but updating java is a one-time affair that should be done as soon as possible.

That being said since you are using a nonstandard bukkit implementation there might be a few more complications when running with this plugin - such as the hybrid perhaps not supporting java 16 at all - at which point you are without help I am afraid.

**Reply by PluginDownloader** [reply]
I tried Java 16 and higher, but nothing. I guess it wont just work. Thank you tho for answering and helping. Appreciate it !

**Reply by Geolykt** [reply]
what is the error message when you use Java 16?

**Reply by PluginDownloader** [reply]
I dont think there is an Error or i havent noticed one. its just the same thing, the plugin wont load

**Reply by Geolykt** [reply]
If the plugin doesn't load there must be an error - I am not aware of anything inside of this plugin that would swallow fatal errors

**Reply by PluginDownloader** [reply]
Maybe i got some weird thingy blocking or something. ill check it out :D! At one point i had hard time to make my server working, kept crashing etc

**Comment by PluginDownloader** [comment, marked as answer]
I think its bc of pixelmon mod

**Reply by PluginDownloader** [reply]
Pixelmon needs like JAVA 8/11, but the plugin needs 16. i guess i found the problem haha. THANKS for helping this slow brain novice :D! 
and sorry for wasting ur time as well ^^
